packages/main/test/samples/StepInput.sample.html

```diff
@@ -0,0 +1,97 @@
+<style>
+	.shorter {
+		width: 300px;
+	}
+</style>
+
+<header>
```

Comment:
You need this "Since marker" so the version is displayed in the API ref next to the StepInput title

```html
<header class="component-heading">
	<h2 class="control-header">StepInput</h2>
	<div class="component-heading-since">
		<span><!--since_tag_marker--></span>
	</div>
</header>
```

Reply:
Done!
* Playground sample fixed.
* valueStateMessage fixed